### ISSUE TITLE
Authentication PIN

### ISSUE THREAD

**Issue by Macrox14** [trakt/Plex-Trakt-Scrobbler#580]
![image](https://user-images.githubusercontent.com/31378074/98900908-bf5eb180-2480-11eb-9a9b-f82738805ea1.png)
Does this work? Because I put the PIN, then saved and nothing happens. And in my trakt account doesn't appears any new app linked

![image](https://user-images.githubusercontent.com/31378074/98901003-f765f480-2480-11eb-9eff-d672e018275f.png)
Also this never changes

**Comment by jbenleo** [comment]
Same issue here.

**Comment by CBarbas** [comment]
Hi,

I have format my PC before some days ago, I was straggling to make this work and some how it has, I am not sure what I have done, I am saying that because if I remember correct I have done whats you describing and it has worked.
Yesterday I have format my PC again and since then it is impossible to activate the application again.

**Comment by sasagr** [comment]
same here

**Comment by sasagr** [comment]
any feedback here? Is the plugin working?

**Comment by CBarbas** [comment]
Hi,

yes it working but from my experience you will have to try many times and ignore the message "We can't find code ******** Your device should be displaying an 8 character code.".
Although the above message the activation works but with a delay, sometimes big.

I am just describing what was happened to me and you will see it here "https://trakt.tv/oauth/authorized_applications" when you realize it is working but after a big delay, the "connected on" "day/time" to be precise.


I hope it helps!

**Comment by sasagr** [comment]
I m following the steps as also described above from @Macrox14 and I m not getting any error but I m not getting also any confirmation that plex has been connected to trakt. I have tried many time and after almost 24 hours I still don't see anything new in the authorised applications. I m also curious to know where are you getting the message "We can't find code.."

**Comment by CBarbas** [comment]
The message

appears when you click here "https://trakt.tv/activate" by putting you PIN and then "continue".

Here are the instructions "https://support.trakt.tv/knowledgebase/articles/885516-how-do-i-connect-my-media-center".

**Comment by sasagr** [comment]
I did not know I had to go there (https://trakt.tv/activate). This step was not clear.
Anyway.. now I did the following:
https://trakt.tv/pin/478
got the PIN
pasted in the trakt.tv settings page pf the plugin
pasted in https://trakt.tv/activate and got the error you mentioned
I will now wait to see if I m going to see anything new in the authorised applications

**Comment by sasagr** [comment]
update... There is nothing in the authorized applications :(

**Comment by CBarbas** [comment]
Quick question,

is your plugin update, I mean latest version, it might be also important.
I think the best option here is to install the plugin via the WebTools following their guide from here "https://github.com/trakt/Plex-Trakt-Scrobbler/wiki/Installation---Windows".
Then follow the activation steps again and again till it works.

I dont know what else to say!

**Comment by sasagr** [comment]
I installed it just last week so I think it is the latest version. My plex server is not on a windows machine so I m not sure I can follow the installation guide you posted. I instead followed this one, https://github.com/trakt/Plex-Trakt-Scrobbler/wiki/Installation---Debian-%28Terminal%29#install-manual
and I must say I did not have any issue with the installation, as a matter of fact I see the plugin installed. The issue is purely the activation! Also tried using the configuration link but I have the same issue with this one, https://github.com/trakt-for-plex/configuration/issues/56
so it does not look like I m only the only one...

**Comment by CBarbas** [comment]
Oh OK,

yeah many people with defferent platforms have problems with activation as I have reliazed by internet searching!

**Comment by marine1988** [comment]
yes activation is not working for me anyone knows how to fix it?

**Comment by michaelblong** [comment]
Same Issue. I had it up and running. I formatted and reinstalled windows and tried to get this running... No Luck.

**Comment by michaelblong** [comment]
So after trying to get it working for a few days, I had given up. I signed up for Trakt VIP today so that I could use webhooks. I then realized that my attempt from the night before finally activated. A show that had recorded today was collected in Trakt.
